README.md

```diff
@@ -49,6 +49,8 @@ DISABLE_UPDATE_CHECKER=true
 ## Command Help
 
 ```
+Incorrect Usage: flag: help requested
+
```

Comment:
Nope you did something wrong if you have this message:
```suggestion
```

Reply:
I have the same output with the scalingo binary of production

Reply:
I remove this line from the README but I invite you to try with your local binary and create an issue if you have the same output

Reply:
It's because the command is `scalingo help` ;)

Reply:
oups :sweat_smile: